pkg/api/controller_test.go

```diff
@@ -4854,6 +4854,10 @@ func TestDistSpecExtensions(t *testing.T) {
 		err = json.Unmarshal(resp.Body(), &extensionList)
 		So(err, ShouldBeNil)
 		So(len(extensionList.Extensions), ShouldEqual, 1)
+		So(len(extensionList.Extensions[0].Endpoints), ShouldEqual, 1)
```

Comment:
Add the test for the actual endpoint match here?

Reply:
Yes, I will add.